pages/cathocomponents.js

```diff
@@ -12,130 +27,122 @@ import {
-        }}
-      />
-      <BarChart
-        data={[
-          {
-            name: "Jan",
-            value: 4000,
-            tooltip: "R$ 4000",
-          },
-        ]}
-      />
-      <CommentsBox><p>Teste</p></CommentsBox>
-      <DayPicker />
-      <FooterSimple />
-      <HorizontalMenu
-        items={[
-          {
-            name: "Buscar Vagas",
-            url: "https://google.com",
-            iconName: "favorite_border",
-            isHighlighted: false,
-          },
-        ]}
-      />
-      <JobCard
-        jobID="123"
-        jobURL="https://www.catho.com.br"
-        name="Test"
-        company={{
-          name: "Catho",
-          avatar: undefined,
-          isConfidential: false,
-          stamps: [
-            {
-              icon: "person",
-              description: "",
-            },
-          ],
-        }}
-        salaryRange="test"
-        benefits={[
-          {
-            name: "Alimentação",
-            icon: "receipt",
-          },
-        ]}
-        publishDate={new Date(2022, 4, 10, 13).getTime()}
-        updateDate={new Date(2022, 5, 10, 13).getTime()}
-        locations={[
-          {
-            name: "test",
-            quantity: 15,
-            url: "https://www.catho.com.br",
-          },
-        ]}
-        quantityTotal={10}
-        description="test"
-        isPCD={false}
-        isExtended={false}
-        isCompatible={false}
-        isFavorite={false}
-        cta={{
-          type: "DEFAULT",
-          dateSent: new Date(2022, 6, 10, 13).getTime(),
-          timeToUmblock: new Date(2022, 6, 10, 14).getTime(),
-          subscribeLink: "https://www.catho.com.br",
-        }}
-      />
-      <JobNotificationList
-        data={[
-          {
-            id: "123",
-            title: "Designer de Produtos, UX Design",
-            qtde: 290,
-            city: [
+      <div class="ColoredBackground ComponentsContainer">
+        <div>
+          <h4 class="TextFormattingH4">Footers</h4>
+          <h6>These are footers for all Catho applications.</h6>
+        </div>
+        <div class="MarginTopBottom">
+          <h5 class="TextFormattingH5">Complete Footer Version with TagList</h5>
+          <FooterResponsive
+            gtmPrefix="B2C-google-tag-prefix"
+            tagsListContent={[
+              {
+                items: candidatesMockContent,
+                label: "Candidates",
+              },
+              {
+                items: companiesMockContent,
+                label: "Companies",
+              },
+              {
+                items: educationMockContent,
+                label: "Education",
+              },
+              {
+                items: privateMockContent,
+                label: "Privates",
+              },
               {
-                name: "São Paulo",
-                uf: "SP",
+                items: extraMockContent,
+                label: "Extra",
               },
```

Comment:
What do you think about passing all these props to a mock object like it was done in jobCard?